engine/src/mac-core.mm

```diff
@@ -557,6 +566,11 @@ - (NSApplicationPrintReply)application:(NSApplication *)application printFiles:(
 @end
 
 ////////////////////////////////////////////////////////////////////////////////
+enum MCPlatformSystemAppearance
```

Comment:
Enums should go in platform.h - perhaps in the 'System properties' section (maybe after the MCPlatformSystemProperty enum).